phpfarm/docker.sh

```diff
@@ -17,14 +17,23 @@ do
 
     # compile the PHP version
     ./compile.sh $VERSION
+
+    # PHP 5.1 names both CLI and CGI executables 'php'. Rename and move them to
+    # correct dirs. See http://php.net/manual/bg/features.commandline.php
+    if [ "$V" == "5.1" ]; then
+        post='custom/post-install-5.1.6.sh'
+        echo "Running commands from '$post'"
+        /bin/bash "$post" "$VERSION"
+    fi
```

Comment:
hmm actually it should already do this as you named the file as phpfarm expects... why calling it from docker.sh again?